tree/readspeed/README.md

```diff
@@ -0,0 +1,70 @@
+# rootreadspeed
+
+`rootreadspeed` is a tool used to help identify bottlenecks in root analysis programs
+by providing an idea of what throughput you can expect when reading ROOT files in
+certain configurations.
+
+It does this by providing information about the number of bytes read from your files,
+how long this takes, and the different throughputs in MB/s, both in total and per thread.
+
+
+## Compressed vs Uncompressed Throughput:
+
+Throughput speeds are provided as compressed and uncompressed - ROOT files are usually
+saved in compressed format, so these will often differ. Compressed bytes is the total
+number of bytes read from TFiles during the readspeed test (possibly including meta-data).
+Uncompressed bytes is the number of bytes processed by reading the branch values in the TTree.
+Throughput is calculated as the total number of bytes over the total runtime (including
+decompression time) in the uncompressed and compressed cases.
+
+
+## Interpreting results:
+
+### There are three possible scenarios when using rootreadspeed, namely:
+
+  - The 'Real Time' is significantly lower than your own analysis runtime.
+    This would imply your actual application code is dominating the runtime of your analysis,
+    ie. your analysis logic is taking up the time.
+    The best way to decrease the runtime would be to optimize your code, attempt to parallelize
+    it onto multiple threads if possible, or use a machine with a more performant CPU.
```

Comment:
Maybe add "Consider using RDataFrame with `ROOT::EnableImplicitMT()` and / or sharing your issue on https://root-forum.cern.ch"?